src/lib/functions.ts

```diff
@@ -224,13 +222,5 @@ function createReplies(functionReply: GleeFunctionReturnReply, message: GleeMess
     replyChannel = channel
   }
 
-  const sendOperations = replyChannel.operations().filterBySend()
-
-  if (!sendOperations || sendOperations.length === 0) {
-    const warningMsg = `No 'send' operations defined for channel '${replyChannel.id()}'. Ensure your AsyncAPI file defines a 'send' operation for this channel to enable message replies. As a result, no reply will be sent for the channel.`
-    logWarningMessage(warningMsg)
-    return []
-  }
-
-  return sendOperations.map(operation => new GleeMessage({ ...functionReply, channel: replyChannel.id(), request: message, operation, connection: message.connection }))
+  return new GleeMessage({ ...functionReply, channel: replyChannel.id(), request: message, operation, connection: message.connection })
```

Comment:
I think we have to create an issue for this. We still need to check but not just on "send" operations but also on replies of a "receive" operation.